### ISSUE TITLE
Follow-up issue for card group title field.

### ISSUE THREAD

**Issue by trackleft** [az-digital/az_quickstart#2203]
## Motivation
It was discussed that we do the follow-up task mention in this issue  during 2022-02-01 PR meeting.

- [ ] Add new title field to card preview
- [ ] Update demo content to use title instead of separate text paragraph
- [ ] Update the Quickstart Documentation to mention the title field
- [ ] Possibly move the title field above the card settings.

## Blocked by
https://github.com/az-digital/az_quickstart/pull/2192

### Roles and Permissions considerations
A clear and concise description of how each of the following roles would be impacted by this change:
- Anonymous user
- Authenticated user
- Content editor
- Content administrator
- Administrator
